### DISCUSSION TITLE
Thanks for the Whisper UI!

### DISCUSSION THREAD

**Discussion by Azar42** [hayabhay/whisper-ui#33] [General]
Just wanted to say thanks for making this project; I've been following other implementations of Whisper in the hopes that someone will make an UI as robust as [Otter.ai](http://www.otter.ai) that I can run locally. [MacWhisper](https://goodsnooze.gumroad.com/l/macwhisper) looks great, but alas I'm on Windows -- so when I found whisper-ui, I realized I could get it running on Linux Subsystem for Windows. After a bit of fiddling, it's up and running!

**Comment by hayabhay** [comment]
Thanks! Given the latent demand for an open-source UI, I'm working on a proper web application for the 2.x version that should make things more flexible. Stay tuned!
